src/malloc/dlmalloc/malloc.c

```diff
@@ -846,6 +846,29 @@ extern "C" {
 #define dlindependent_calloc   independent_calloc
 #define dlindependent_comalloc independent_comalloc
 #define dlbulk_free            bulk_free
+#else /* USE_DL_PREFIX */
+
+#ifdef __CHEERP__
+#define CHEERP_ASMJS_ATTRIB __attribute((cheerp_asmjs))
```

Comment:
Is this actually used? I don't see it

Reply:
It was used previously, forgot to remove. Making new commit